### PR TITLE
feat: add openapi app plugin to catalog modules

### PR DESCRIPTION
**Pull request by fernandoqueirozzup**

### CODE REVIEW

**Review by ricardojsilva — DISMISSED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -28,3 +28,6 @@
 [submodule "grpc-server-app-kt-plugin"]
 	path = grpc-server-app-kt-plugin
 	url = https://github.com/stack-spot/grpc-server-app-kt-plugin
+[submodule "openapi-gradle-app-plugin"]
+	path = openapi-gradle-app-plugin
+	url = https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin
```
Comment (LG95) [inline .gitmodules:33]: Please run `git submodule add https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin` to fully configure this.

Comment (LG95) [inline .gitmodules:33]: Please run `git submodule add https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin` to fully configure this.

**Review by LG95 — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (fernandoqueirozzup) [reply .gitmodules]: Done

**Review by ricardojsilva — APPROVED** [review]

File: .gitmodules [hunk 2]
```diff
@@ -28,6 +28,6 @@
 [submodule "grpc-server-app-kt-plugin"]
 	path = grpc-server-app-kt-plugin
 	url = https://github.com/stack-spot/grpc-server-app-kt-plugin
-[submodule "openapi-gradle-app-plugin"]
-	path = openapi-gradle-app-plugin
+[submodule "uxservices-kotlin-openapi-gradle-app-plugin"]
+	path = uxservices-kotlin-openapi-gradle-app-plugin
 	url = https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin
```
Comment (LG95) [inline .gitmodules:33]: This needs a public mirror.

**Review by LG95 — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 2 above)
Comment (fernandoqueirozzup) [reply .gitmodules]: Done.

**Review by LG95 — DISMISSED** [review]
Why add a `gitignore`?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+gradle-app.setting
+
+# Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
+!gradle-wrapper.jar
+
+# Cache of project
+.gradletasknamecache
+
+# # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
+# gradle/wrapper/gradle-wrapper.properties
```
Comment (LG95) [inline .gitignore:45]:
```suggestion
```

**Review by LG95 — APPROVED** [review]

**Review by ricardojsilva — APPROVED** [review]

**Pull request merged**